libvips/foreign/heifsave.c

```diff
@@ -496,6 +519,7 @@ vips_foreign_save_heif_init( VipsForeignSaveHeif *heif )
 	heif->Q = 50;
 	heif->compression = VIPS_FOREIGN_HEIF_COMPRESSION_HEVC;
 	heif->speed = 5;
+	heif->subsample_mode = VIPS_FOREIGN_SUBSAMPLE_AUTO;
```

Comment:
This is a breaking change as the current default is to always use 4:2:0 subsampling.

Reply:
Ah true. Though I minor one I suppose.